.travis.yml

```diff
@@ -2,12 +2,10 @@ language: python
 dist: xenial
 
 python:
-    - "2.7"
     - "3.4"
     - "3.5"
     - "3.6"
     - "3.7"
-    - "pypy2.7-6.0"
     - "pypy3.5-6.0"
```

Comment:
looks like Travis [supports pypy 3.6.1](https://docs.travis-ci.com/user/languages/python/#pypy-support) now, and even has it as the default?

I don't know how any of this works.  I have Ubuntu Bionic (i.e. newer than the Xenial on Travis) on my local machine and I don't see any version of pypy3 in my apt repos. And [this page](https://pypy.org/download.html) says pypy 3.6 is still beta.